plugins/generator-1.16.5/forge-1.16.5/utils/procedures.java.ftl

```diff
@@ -46,6 +46,52 @@
     ))
 </#macro>
 
+<#macro procedureToRetvalTypeCode name dependencies type customVals={}>

```

Comment:
Why is this change needed? I think you can use the existing condition code for redstone conditions too.

Reply:
Redstone conditions need to output an integer. Therefore, I need to specify a type.

Reply:
I see